pkg/service/backup/restore_worker_index.go

```diff
@@ -32,7 +32,7 @@ type indexWorker struct {
 
 	bundles      map[string]bundle       // Maps bundle to it's ID
 	bundleIDPool chan string             // IDs of the bundles that are yet to be restored
-	resumed      bool                    // Set to true if current run has already skipped all tables restored in previous run
+	continuation bool                    // Set to false if current run has already skipped all tables restored in previous run
```

Comment:
I'm not really sure about this name change - although it is surely a good direction.
Worker uses this variable to check if it has already skipped all Locations/Manifests/FilesMeta restored entirely in the previous run.

So it is strange to me to set `continuation` to false after skipping all previously processed tables - it sounds more like the `continue` from `RestoreTarget` that does not change during restore procedure.
Maybe calling it sth like `isPrevSkipped` would make more sense?